chatbox.html

```diff
@@ -152,7 +153,12 @@
 
 <main>
   <header>
-    <h2>Your inbox...</h2>
+    <div>
+      <h2>Your inbox...</h2>
+    </div>
+    <div>
+      <span>Needs Chrome v63+</span>
```

Comment:
I'd prefer to do a feature detect instead and fill this span with "Your browser doesn't support CSS overscroll-behavior" if it fails.